rele/middleware.py

```diff
@@ -0,0 +1,56 @@
+import importlib
+
+_middlewares = []
+
+
+def register_middleware(paths):
+    global _middlewares
```

Comment:
Perhaps this should be `_rele_middlewares`. That way there is even a smaller chance of conflicting with other packages. What do you think?

Reply:
`_middlewares` is defined in the module. What happens outside it doesn't have any effect (unless we did a star import from somewhere, that is)